guides/next-version.md

```diff
@@ -0,0 +1,18 @@
+# Next Version
+
+When the Cypress CLI and binary are built in CI, their versions are determined by the script in [`/scripts/get-next-version.js`](../scripts/get-next-version.js).
```

Comment:
It's worth mentioning the CLI & Binary versions are in sync/ will match.

Reply:
```suggestion
When the Cypress CLI and binary are built in CI, the version number they share is determined by the script in [`/scripts/get-next-version.js`](../scripts/get-next-version.js).
```